internal/transformations/desugarer_test.go

```diff
@@ -14,4 +14,4 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package jsonnet
+package transformations
```

Comment:
I'm open to suggestions for a better package name. The rationale behind "transformations" is that it transforms the AST.

Reply:
Perhaps just "transform" would read better for importing code: `transform.SnippetToRawAST`.

Reply:
I went with `program`.